src/components/Graph/GraphFilter.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import Legend from '../Graph/Legend';
+import Scenarios from '../Filters/Scenarios';
+import Indicators from '../Filters/Indicators';
+import Overlays from '../Filters/Overlays';
+import SeverityContainer from '../Filters/SeverityContainer'
+import Sliders from '../Filters/Sliders';
+
+
+class GraphFilter extends Component {
+    handleScenarioClick = (i) => {
+        this.props.onScenarioClick(i);
+    }
+
+    handleButtonClick = (i) => {
+        this.props.onButtonClick(i);
+    }
+    
+    handleConfClick = () => {
+        this.props.onConfClick();
+    }
+
+    handleSeveritiesClick = (i) => {
+        this.props.onSeveritiesClick(i);
+    }
+
+    handleSeveritiesHover = (i) => {
+        this.props.onSeveritiesHover(i);
+    }
+
+    handleSeveritiesHoverLeave= () => {
+        this.props.onSeveritiesHoverLeave();
+    }
+
+    handleStatSliderChange = (i) => {
+        this.props.onStatSliderChange(i);
+    }
+
+    handleDateSliderChange = (i) => {
+        this.props.onDateSliderChange(i);
+    }
+
+    render() {
+        return (
+            <div>
+                <Legend />
+                <div className="param-header">SCENARIOS</div>
+                <Scenarios
+                    SCENARIOS={this.props.SCENARIOS}
+                    scenario={this.props.scenario}
+                    scenarioList={this.props.scenarioList}
+                    onScenarioClick={this.handleScenarioClick}
+                />
+                <Indicators
+                    stat={this.props.stat}
+                    onButtonClick={this.handleButtonClick}
+                />        
+                <Overlays 
+                    showConfBounds={this.props.showConfBounds}
+                    onConfClick={this.handleConfClick}
+                /> 
+                <SeverityContainer
+                    severityList={this.props.severityList}
+                    scenarioList={this.props.scenarioList}
+                    onSeveritiesClick={this.handleSeveritiesClick}
```

Comment:
unless you plan to have extra logic at in this component when `this.handleSeveritiesClick` is called, you can pass `this.props.handleSeveritiesClick`. That way you don't have to make all those methods from line 11-41